core/src/window_service.rs

```diff
@@ -62,19 +62,14 @@ fn process_blobs(blobs: &[SharedBlob], blocktree: &Arc<Blocktree>) -> Result<()>
         }
     }))?;
 
-    for blob in blobs {
```

Comment:
should blocktree have better apis for this, i.e. be able to tell for itself the disposition of a blob instead of having window_service do the filtering?

Reply:
Made an issue https://github.com/solana-labs/solana/issues/4724